gandy/tests/test_models/test_gans.py

```diff
@@ -0,0 +1,200 @@
+"""Testing functions for UncertaintyModel gan class."""
+
```

Comment:
Functionally only things to consider are maybe adding more tests of failure modes, eg. if you can foresee some weird user input screwing things up, write a test to make sure that if such an input is passed, your function catches it and raises an error. Overall though, functionally very good, flow makes sense and seems to be checking important things.

Most of my comments are purely syntactical where the code will break. Didn't repeat comments if I saw the same thing later.